backend/__tests__/posts.js

```diff
@@ -402,4 +402,121 @@ describe('Posts', () => {
         */
         describe('when deleting a parent post', () => {})
     })
+
+    describe('POST /posts/:id/like', () => {
+        describe('when user like post in a valid way', () => {
+            it('should return response code of 201', async () => {
+                const user1 = await models.users.create({
+                    username: 'gandalf',
+                    nickname: 'gandalf',
+                    email: 'gandalf@gandalf.com',
+                    password: 'password',
+                })
+
+                const authToken = Authentication.generateAuthToken(user1)
+
+                const createPostResponse = await request(server)
```

Comment:
I made this same exact mistake, but we're not supposed to use other endpoints in the tests. What we have to do is to directly create a post by manipulating the model. You can check the updated version of my tests in my PR to get an idea of what I mean.

Reply:
Ok. 
Thanks for the great feedback !